### DISCUSSION TITLE
Making it Routine

### DISCUSSION THREAD

**Discussion by julieadair73** [CivicActions/accessibility#613] [Q&A] [answered]
What is the best way to make accessibility second nature with every epic or ticket?

**Comment by dmundra** [comment, marked as answer]
@julieadair73 my vote would be to inject into the agile process with these ideas (taken from https://accessibility.civicactions.com/agile)

> We encourage accessibility in every part of agile processes. Examples, adding inclusive user archetypes to design sprints, adding accessibility tests to the definition of done, demonstrating how a product is accessible in sprint reviews, prioritizing accessibility goals in retrospectives, delivering accessible product increments, and much more.
